modules/ivc_champva/app/models/ivc_champva/ves_request.rb

```diff
@@ -1,5 +1,21 @@
 # frozen_string_literal: true
 
+##
+# Function to get all instance variables from a class and convert them
+# to a hash, converting names from snake_case to camelCase symbols.
+# Also runs `to_hash` on any existing top-level `address` properties.
+#
+def instance_vars_to_hash(instance)
+  # Create hash where keys are the instance variables with '@' removed and
+  # converted from snake_case to camelCase
+  # e.g.: '@phone_number' -> :phoneNumber
+  hash = instance.instance_variables.to_h do |name|
```

Comment:
I think using `.instance.instance_variables.each_with_object({})` is cleaner here because it allows you to add logic inside the loop. Also using `.delete_prefix`, which is more explicit than `tr('@', '')`.

Something like

```
instance_vars_hash = instance.instance_variables.each_with_object({}) do |var, hash|
  key = var.to_s.delete_prefix('@').camelize(:lower).to_sym
  value = instance.instance_variable_get(var)
  hash[key] = value
end

if instance.respond_to?(:address) && instance.address.respond_to?(:to_hash)
  instance_vars_hash[:address] = instance.address.to_hash
end

instance_vars_hash.compact
  ```